spinoso-string/src/lib.rs

```diff
@@ -1220,6 +867,34 @@ impl String {
     }
 }
 
+// Additional IntoIterator iterator
+impl IntoIterator for String {
+    type Item = u8;
+    type IntoIter = IntoIter;
+    /// Returns an iterator that moves over the remaining bytes of a slice
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use spinoso_string::String;
+    ///
+    /// let s = String::from("abc");
+    ///
+    /// let mut iterator = s.into_iter();
+    ///
+    /// assert_eq!(iterator.next(), Some(b'a'));
+    /// assert_eq!(iterator.next(), Some(b'b'));
+    /// assert_eq!(iterator.next(), Some(b'c'));
+    /// assert_eq!(iterator.next(), None);
+    /// ```
+
```

Comment:
nit to fix the whitespace

```suggestion

    /// Returns an iterator that moves over the remaining bytes of a slice
    ///
    /// # Examples
    ///
    /// ```
    /// use spinoso_string::String;
    ///
    /// let s = String::from("abc");
    ///
    /// let mut iterator = s.into_iter();
    ///
    /// assert_eq!(iterator.next(), Some(b'a'));
    /// assert_eq!(iterator.next(), Some(b'b'));
    /// assert_eq!(iterator.next(), Some(b'c'));
    /// assert_eq!(iterator.next(), None);
    /// ```
```